components/organisms/ResetPassword/ResetPassword.tsx

```diff
@@ -69,8 +69,7 @@ export const ResetPassword = () => {
         Asignar nueva contraseña
       </h2>
       <p className={`${styles["form-reset_paragraph"]}`}>
-        Al terminar, te enviaremos a iniciar sesión de nuevo con tu nueva
-        contraseña
+        Al terminar, Inicia sesión con tu nueva contraseña.
```

Comment:
Oh, sorry, my bad... Use i instead of I please

```suggestion
        Al terminar, inicia sesión con tu nueva contraseña.
```

Reply:
Dani is already corrected